Ruby-Project/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, book)
+    rental = Rental.new(date, book)
+    @rental << rental
+    person.add_rental(rental)
+  end
```

Comment:
For this method, the only thing that's necessary is to create a new rental. Its creation will automatically add the rental to the rentals list for the Book and Person objects. 
Also the parameters passed should be date and person. For the book we'll be passing `self`. Something like this should do: 
```
def add_rental(person, date)
    Rental.new(date, self, person)
end
```